lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -829,15 +829,16 @@ def upload_grades_csv(_xmodule_instance_args, _entry_id, course_id, _task_input,
             course_grade.letter_grade,
             student.id in whitelisted_user_ids
         )
-        if certificate_info[0] == 'Y' and course_grade.letter_grade is None and student.id not in whitelisted_user_ids:
```

Comment:
Grade reports were regenerated (with students who were failing marked as "Y") and the log was not triggered. So I have made the log less specific so that it triggers for everyone who is marked "Y". This is just for the time being so that we can find the cause of the incorrect grade reports.